client/src/component/qa/AddAnswer.jsx

```diff
@@ -1,41 +1,122 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import axios from 'axios';
 
 const Container = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 100vh;
+position: fixed;
+z-index: 100;
+top: 0;
```

Comment:
these are fine but need an indentation for the css ie.
const Container = styled.div`
    position: fixed;
    z-index: 100;
    top: 0;
`

Reply:
oops that didn't render correctly, but just add a tab before each line